nft/nfts.yaml

```diff
@@ -998,7 +999,7 @@ components:
       in: query
     pageSize:
       name: pageSize
-      description: 'String - Number of NFTs to be returned per page. Defaults to 100. Max is 100. NOTE: Only supported on Ethereum Mainnet and Goerli, Polygon Mainnet and Mumbai, Arbitrum Mainnet and Goerli, and Optimism Mainnet and Goerli.'
+      description: 'Number of NFTs to be returned per page. Defaults to 100. Max is 100.'
```

Comment:
@ayang-rkjs can we set the default field here to 100?

Reply:
done!